draft-ietf-quic-recovery.md

```diff
@@ -414,6 +445,15 @@ acknowledged.  DetectLostPackets is called every time there is a new largest
 packet or if the loss detection alarm fires the previous largest acked packet is
 supplied.
 
+### Handshake Packets
+
+The receiver MUST not trust an unencryped ack for encrypted packets.  The receiver
```

Comment:
@martinduke is right to ask, note that not just the acknowledgment, but the entire packet is now suspect.  As we discussed elsewhere, I think that the entire packet needs to go away if it contains suspicious content.